clients/src/main/java/org/apache/kafka/clients/admin/CreateTopicsResult.java

```diff
@@ -64,6 +64,7 @@ public KafkaFuture<Void> all() {
      * If broker returned an error for topic configs, throw appropriate exception. For example,
      * {@link org.apache.kafka.common.errors.TopicAuthorizationException} is thrown if user does not
      * have permission to describe topic configs.
+     * Note that it is possible to obtain type and documentation with null values even if they are not defined
```

Comment:
The protocol does not carry the `type` and `documentation`, so those fields must be "null".

Reply:
Thank you, I have refined the text. Please take a look.

Reply:
I further simplified the description.